v1.0/v1.0/touch.cwl

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env cwl-runner
+cwlVersion: v1.0
+class: CommandLineTool
+
+hints:
+  DockerRequirement:
+    dockerPull: debian:stretch-slim
```

Comment:
IMO it will be better to use `requirements` rather than `hints` because the behavior of `hints` depends on the platform and environment.

Reply:
But for this particular conformance test the use of Docker or not is not essential, so I would leave it as `hints`